chainer/links/connection/n_step_lstm.py

```diff
@@ -50,6 +50,22 @@ def __call__(self, hx, cx, xs, **kwargs):
             xs (list of ~chainer.Variable): List of input sequences.
                 Each element ``xs[i]`` is a :class:`chainer.Variable` holding
                 a sequence.
+
+        Returns:
+            tuple: This function returns a tuple containing three elements,
+            ``hy``, ``cy`` and ``ys``.
+
+            - ``hy`` is an updated hidden states whose shape is the same as
+              ``hx``.
+            - ``cy`` is an updated cell states whose shape is the same as
+              ``cx``.
+            - ``ys`` is a list of :class:`~chainer.Variable` . Each element
+              ``ys[t]`` holds hidden states of the last layer corresponding
+              to an input ``xs[t]``. Its shape is ``(B_t, N)`` for
+              uni-directional LSTM and ``(B_t, 2N)`` for bi-directional LSTM
```

Comment:
``B_t`` is the length of the t-th sequence, isn't it?

Reply:
Thank you for your comment.
Yes, `B_t` is the length of t-th sequence.
I found that the document of `n_step_lstm` function contains also the wrong comment.
https://github.com/chainer/chainer/blob/v3.4.0/chainer/functions/connection/n_step_lstm.py#L296